### ISSUE TITLE
Large all_fasta.loc causes pages that read the table to become unresponsive

### ISSUE THREAD

**Issue by innovate-invent** [galaxyproject/galaxy#8392]
I have a 2.5Mb all_fasta.loc and looking at the table in the admin page takes forever to load.
Even worse is trying to run any tool with an input that references the table, it times out while loading.

**Comment by mvdbeek** [comment]
Yeah, that's surely not designed for that many entries. How many lines do you have in there, and how is this being used ?

**Comment by innovate-invent** [comment]
There are 13234 records. This is our bacteria genome database.
It is being used as the reference to align scaffolds with strain level specificity.

**Comment by innovate-invent** [comment]
I just tried the API endpoint and it seems snappy. It looks like the javascript is failing to handle the response rather than the backend failing to generate it in a timely manner.
